app/src/androidTest/java/org/mozilla/fenix/helpers/MockWebServer.kt

```diff
@@ -13,7 +13,7 @@ import okhttp3.mockwebserver.MockResponse
 import okhttp3.mockwebserver.MockWebServer
 import okhttp3.mockwebserver.RecordedRequest
 import okio.Buffer
-import okio.source
+//import okio.source
```

Comment:
I needed to make some changes in this file to get the build to compile, not correct but focusing on getting the apk and coming back to fix these errors later